### PR TITLE
Add tokei

### PR DESCRIPTION
**Pull request by dbast**

### CODE REVIEW

File: recipes/tokei/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% set version = "7.0.3" %}
+
+package:
+  name: tokei
+  version: {{ version }}
+
+source:
+  url: https://github.com/Aaronepower/tokei/releases/download/v{{ version }}/tokei-v{{ version }}-x86_64-unknown-linux-gnu.tar.gz  # [linux]
```
Comment (mariusvniekerk) [inline recipes/tokei/meta.yaml]: Thanks for the submission.  
In general conda-forge prefers not to repackage binary artifacts unless there is no other way of building the package.  Could we try at least to build for osx and windows?

Comment (dbast) [reply recipes/tokei/meta.yaml]: Ok. will try that.

**Comment by dbast** [comment]
MacOS build fails with: (signal: 11, SIGSEGV: invalid memory reference) ... searching tells, that this could be potentially related to a too old SDK... the used SDK in that build is MacOSX10.9.sdk ...

Does anybody have any idea how to resolve this?

**Comment by marcelotrevisani** [comment]
@mariusvniekerk is that good to go?

**Comment by dbast** [comment]
Updated the recipe:
- Now builds from source (no binary downloads, also removed the previous commit that did that)
- Builds on all 3 platforms, linux-64, mac-64 and win-64 (due to updates outside of the recipe, CI code or Rust pkg updates)
- License also packaged

@mariusvniekerk Anything else to be done here?

**Pull request merged**